packages/design-system/docs/gettingStarted/usage.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Usage
+next: false
+---
+
+# Usage
+
+## Components
+
+If the design-system is initialized properly, there is no need to import components. For example, to use the `OcButton` component:
+
+```
+<oc-button>
+  Click me!
+</oc-button>
+```
+
+## Utility classes
+
+The design-system does not provide any utility classes. Instead, it uses Tailwind, which is provided by the parent application. Please refer to the official [Tailwind docs](https://tailwindcss.com/docs) for more information.
+
+## Helper methods
+
+The design-system provides a set of helper methods and composables that might be helpful, especially when developing apps or extensions within the OpenCloud ecosystem.
```

Comment:
Ideally here should be a link to the docs for the helper methods, unfortunately they don't exist yet.